verilog/tools/kythe/verilog_kythe_extractor.cc

```diff
@@ -37,13 +37,17 @@ ABSL_FLAG(bool, printkythefacts, false,
 ABSL_FLAG(std::string, output_path, "",
           "File path where to write the extracted Kythe facts in JSON format.");
 
+ABSL_FLAG(std::string, include_dir_paths, "",
+          "Paths of the directorires used to look for included files.");
```

Comment:
`std::vector<std::string>` is ok for now, if we remember `// TODO: support repeatable flag`.

FYI, I've showed the `LineRange` technique to the absl team, and they "don't do this, we can't promise this will always work".  If they ever add logic that implements last-flag-wins by default, that technique will stop working.  I accept we'll have to deal with the problem then.

Reply:
It can also be accomplished by providing a flag-translator wrapper shell script -- solution by adding a level of indirection.   `verible-flags-tool.sh join -- verible-verilog-kythe-extractor -I ... -I ... -I ...`